apiPivotalTest/src/main/java/com/fundacionjala/apiPivotalTest/RequestManager.java

```diff
@@ -0,0 +1,44 @@
+package com.fundacionjala.apiPivotalTest;
+
+import java.util.Map;
+
+import com.jayway.restassured.http.ContentType;
+import com.jayway.restassured.response.Response;
+import com.jayway.restassured.specification.RequestSpecification;
+
+import static com.jayway.restassured.RestAssured.given;
+
+public class RequestManager {
+
+    private static RequestSpecification REQUEST = Authentication.getInstance().getRequestSpecification();
+
+    private RequestManager() {
+    }
+
+    public static Response getRequest(String endpoint) {
+        return given().spec(REQUEST)
+                .when().get(endpoint)
+                .then().contentType(ContentType.JSON)
+                .extract().response();
+
+    }
+
+    public static Response postRequest(String endpoint, Map<String, Object> parameters) {
+        return given().spec(REQUEST).params(parameters)
+                .when().post(endpoint)
+                .then().contentType(ContentType.JSON)
+                .extract().response();
+    }
+
+    public static Response putRequest(String endpoint, Map<String, Object> parameters) {
+        return given().spec(REQUEST).params(parameters)
```

Comment:
@HenrrySalinasJala 
Please share this with everyone. 
with this implementation we don't need JSONObject and JSONParser

Good job!.